app/models/shelter.rb

```diff
@@ -1,6 +1,7 @@
 class Shelter < ApplicationRecord
   validates_presence_of :name, :address, :city, :state, :zip
   has_many :pets, dependent: :delete_all
+  has_many :reviews, dependent: :delete_all
```

Comment:
one to many relationship. A shelter has_many reviews. Clever with the dependency delete too!